src/services/types.ts

```diff
@@ -95,6 +95,9 @@ namespace ts {
         /** Gets a portion of the script snapshot specified by [start, end). */
         getText(start: number, end: number): string;
 
+        /* @internal */

```

Comment:
but other users of the API could have their own implementation of a ScriptSnapshot, and these will not have a getFullText..